core/application/goal-state.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application
+
+import (
+	"time"
+)
+
+type GoalStateStatus struct {
+	Status string     `json:"status" yaml:"status"`
```

Comment:
stuff in core doesn't use serialisation tags
these tags should be used in structs in the CLI/hook command